index.bs

```diff
@@ -39,6 +43,10 @@ spec: mediastream-recording; urlPrefix: https://www.w3.org/TR/mediastream-record
 
 spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
     type: dfn; text: valid mime type; url: valid-mime-type
+
+spec: ecmascript; urlPrefix: https://tc39.es/ecma262/multipage/
+    type: dfn;
+        text: current realm; url: executable-code-and-execution-contexts.html#sec-code-realms
```

Comment:
I don't have anything better to suggest right away but I don't like it that we have to create an anchor for something as common as queueing a task. I'll try to think at better ways to formulate things. In the meantime, that looks good enough ;)

Reply:
Actually, I have something to suggest: reference the actual term defined in ECMAScript at the given fragment: "the current realm record", which can be referenced with `[=the current realm record=]` without having to introduce a new anchor.

Reply:
Done